File: BoardGameTracker.Common/Models/ImportGame.cs
using BoardGameTracker.Common.Enums;

namespace BoardGameTracker.Common.Models;

public class ImportGame
{
    public required string Title { get; set; }
    public int BggId { get; set; }
    public required string ImageUrl { get; set; }
    public GameState State { get; set; }
    public bool HasScoring { get; set; }
    public double Price { get; set; }
    public DateTime AddedDate { get; set; }
}

File: boardgametracker.client/src/models/Games/ImportGame.ts
import type { GameState } from "./GameState";

export interface ImportGame {
	title: string;
	bggId: number;
	state: GameState;
	imageUrl: string;
	checked: boolean;
	inCollection: boolean;
	hasScoring: boolean;
	price: number;
	addedDate: Date;
	lastModified: Date;
}
